src/web/server/app/controllers/auth.ts

```diff
@@ -291,7 +294,21 @@ const rtmpAuthPublish = async (req: Request, res: Response, start: boolean) => {
         );
         const saverResponse = await axios.get(saverUrl);
         if (saverResponse.status === 200) {
-          res.status(200).send('OK');
+          if (start) {
```

Comment:
What's `start` ?

Reply:
start indicates whether the rtmp stream has started or ended

Reply:
yes